package.json

```diff
@@ -15,12 +15,16 @@
 		"dev:angular": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=angular yarn workspace @framework/site dev",
 		"dev:vue": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=vue yarn workspace @framework/site dev",
 		"dev:qwik": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=qwik yarn workspace @framework/site dev",
+		"dev:solidjs": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=solidjs yarn workspace @framework/site dev",
+		"dev:svelte": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=svelte yarn workspace @framework/site dev",
 		"build:system": "yarn workspace @framework/system build",
 		"build:landing": "yarn workspace @framework/landing-site build",
 		"build:react": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=react yarn workspace @framework/site build",
 		"build:angular": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=angular yarn workspace @framework/site build",
 		"build:vue": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=vue yarn workspace @framework/site build",
 		"build:qwik": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=qwik yarn workspace @framework/site build",
+		"build:solidjs": "cross-env SNOWPACK_PUBLIC_FRAMEWORK=solidjs yarn workspace @framework/site build",
```

Comment:
The `solidjs` and `svelte` part may be a mistake? 🙂

Reply:
hmmm, I did a git rebase of the main branch, seems there are other teams working on solidjs.

Reply:
Interesting, maybe you need to do an interactive rebase and drop the solidjs/svelte commits...